src/Samples/TestEmbedded/Directory.Build.targets

```diff
@@ -1,2 +1,121 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Import 
+    Condition="'$(MSBuildProjectExtension)' == '.csproj' and '$(SimulateCsWinRTNugetReference)' == 'true'" 
+    Project="$([MSBuild]::GetPathOfFileAbove('Microsoft.Windows.CsWinRT.targets', '$(MSBuildThisFileDirectory)../../../nuget'))" />
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.targets', '$(MSBuildThisFileDirectory)../../'))" />
+
+  <!-- CsWinRTCheckDependencies is also defined in the src/ targets file, and prevents issues when using TestWinRT.
+       We skip the check here because we need the targets but aren't using TestWinRT.
+  -->
+  <Target Name="CsWinRTCheckDependencies" />
+
+  <!-- We overwrite the targets from Microsoft.Windows.CsWinRT.Embedded.targets
+       They copy WinRT.Runtime sources based on the target platform; the filenames designate what platform they're for. -->
+ 
+  <Target Name="CsWinRTAddEmbeddedRuntime_AnySources"
+          Condition="'$(CsWinRTEmbedded)' == 'true'"
+          DependsOnTargets="CsWinRTPrepareProjection"
+          AfterTargets="CsWinRTGenerateProjection"
+          BeforeTargets="CsWinRTIncludeProjection"> 
+  
+    <PropertyGroup>
+      <EmbeddedAnyFiles>
+        $(WinRTRuntimeSourcesDir)TypeNameSupport.cs;
```

Comment:
Any reason why we can't make this *.cs similar to the cswinrt targets or make use of the cswinrt targets logic here?

Reply:
Yeah the issue is * will grab net5 and netstandard2 versions of files. I tried some fanciness that grabbed them all and then took them out, and I tried string functions in msbuild to ignore net5/netstandard2 filenames, but neither was satisfactory. 

This is definitely something I want a better solution for. If you add a new file to WinRT.Runtime, you need to add it here too.

Reply:
The thing you mention about having to add it here was my concern as it is likely to be forgotten.  What was the issue when you tried out the first approach (i.e. get all winrtruntime sources - your net5 list - your netstandard list)?  Also any reason why we can't just get the CsWinRT nuget targets to do what we want here and just update the property it looks under to be like a new property like WinRTRuntimeSourcesDir and in the case of nuget reference we set it to the current value and in the case of it is building here, we override it similar to CsWinRTPath that you see above.